docs/guide/alerts-guide.md

```diff
@@ -0,0 +1,95 @@
+title: Sematext Alerts User Guide
+description: Sematext Cloud Alerts give you a flexible centralized notification system that lets you manage Alert Rules and Alert Conditions across your whole stack. Use Events to correlate them with Metrics and/or Logs, and see what is happening with your infrastructure.  
+
+Receiving Alerts when your software is misbehaving is crucial for every developer. Good alerting practices include:
+
+- **Alert Rules** that **detect Anomalies** and notify when key **performance indicators spike or drop**
+- **Criteria** that triggers **Alerts for monitored resources**
+- **Thresholds** defined for **Alerts** when selected **metrics reach certain severity levels**
+- **Notification channels** through which the responsible team will be notified about the incident
+
+[Sematext Alerts](../alerts) help you address underlying issues before they affect user experience. There are three types of Alerts. 
+
+- **Threshold** - Alerts that are based on classic thresholds. They are **triggered** when something **crosses a pre-defined threshold**.
+- **Anomaly** - Alerts based on statistical anomaly detection. They are **triggered** when **values** suddenly **change and deviate from the baseline**. 
```

Comment:
Not too much bolding?

Reply:
I quite like it so that someone can skim through it without reading and still understand the core concept.